kokoro/builds/build.sh

```diff
@@ -209,16 +209,17 @@ if [ -n "$1" ]; then
     pushd ${REPO_ROOT} > /dev/null
 
     echo -e "\n\n\nHere is the full list of all things iwyu found:"
-    cat build/iwyu_results.txt
+    cat build/include-what-you-use.log
 
-    echo -e "\n\n\nThe following include problems occur in files this PR touched:"
+    echo -e "\n\n\nThe following is a diff that fixes all problems in files this PR touched:"
```

Comment:
It would be nice if the script would not just output the plain diff, but also the command to apply this diff, such that you can just copy&paste the line.

Reply:
Good idea!

Reply:
Done.